app/controllers/api/v1/admin/dashboard_controller.rb

```diff
@@ -0,0 +1,35 @@
+class Api::V1::Admin::DashboardController < Api::V1::ApplicationController
+  include AdminAuthorization
+
+  def index
+    orders = Order.where.not(status: :cart)
+              .where.not(created_at: nil)
+              .order(created_at: :desc)
+              .includes(:user)
+
+    pending_orders = orders.select { |order| order.status == "pending" }
+    completed_orders = orders.select { |order| order.status == "completed" }
+    delivered_orders = orders.select { |order| order.status == "delivered" }
+    @orders = pending_orders + completed_orders + delivered_orders
+
+    render json: @orders
+  end
+
+  def mark_as_handled
+    @order = Order.find(params[:order_id])
```

Comment:
you don't need to set global variables here, just `order` instead of `@order`. Please update the same everywhere in API controllers